docs/usage.md

```diff
@@ -97,6 +97,42 @@ $image-path: 'img';
 @import 'identity-style-guide/dist/assets/scss/styles';
 ```
 
+If you're using Sprockets and precompiling assets you'll need to update your
+Sprockets manifest to include the images for production:
+
+```js
+// app/assets/config/manifest.js
+
+//= link_tree ../../../node_modules/identity-style-guide/dist/assets/img
+```
+
+Unfortunately, this results in the assets being saved under paths that include
+everything after the `node_modules` directory, so a helper method is useful in
+cleaning up the views:
+
+```ruby
+# app/helpers/assets_helper.rb
+
+module AssetsHelper
+  def identity_asset_path(path)
```

Comment:
"identity" seems a little broad to reference the design system assets specifically, though I'm not sure I have a better alternative to offer. `lgds_asset_path` works but the abbreviation may not be obvious to everyone. `login_design_system_asset_path` or variations thereof can work and are descriptive, but exceedingly so.

Reply:
Maybe `login_design_asset_path` to keep it from getting too long?

Reply:
> Maybe `login_design_asset_path` to keep it from getting too long?

Any of the above would be fine with me. Even keeping it as-is could work, considering it's just an example and could be customized specific to the project where it's implemented.